user_events.py

```diff
@@ -420,17 +432,34 @@ def user_an_event_edit(id):
                     if old_sub_events is None or new_sub_event not in old_sub_events:
                         new_added_subevent = None
                         if 'id' in new_sub_event:
+                            found = False
```

Comment:
It might be better to add some brief comments to explain these `if` conditions. We could easily forget this in the near future. Thanks.

Reply:
Thanks.